gwsumm/plot/core.py

```diff
@@ -324,6 +324,8 @@ def pid(self):
             filts = "".join(map(str,
                 [getattr(c, 'filter', getattr(c, 'frequency_response', ''))
                  for c in self.channels]))
+            if self.filterstr:
+                filts = "".join([filts,self.filterstr])
```

Comment:
Right, I saw that was possible but thought that having the filter string included in the md5 hash was cleaner